spec/presenters/your_details_form_presenter_spec.rb

```diff
@@ -1,37 +1,29 @@
 RSpec.describe Wpcc::YourDetailsFormPresenter do
   subject do
-    described_class.new(
-      your_details_form: your_details_form,
-      translator: translator
-    )
+    described_class.new(your_details_form, view_context: view_context)
   end
+
   let(:your_details_form) { Wpcc::YourDetailsForm.new }
-  let(:translator) do
-    ->(value) { value.sub(/wpcc.details.options.frequency./, '') }
-  end
+  let(:view_context) { double(:view_context) }
```

Comment:
if you add ActionController::Base.new.view_context you don't need any stub or mock during the specs.